spec/features/express_donations/email_one_click_spec.rb

```diff
@@ -20,7 +20,6 @@ def register_member(member)
     fill_in 'password', with: 'password'
     fill_in 'password_confirmation', with: 'password'
     click_button 'Register'
-    expect(page).to have_content("window.location = '/a/foo-bar/follow-up?member_id=#{member.id}'")
```

Comment:
When you go to `/new` you can pass an optional param `follow_up_url`. That param is stored in the session and then used after a successful registration. If the param is not present, the user is redirected to the home page.